.github/workflows/build-test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                cloud: [ 'AWS' ]
+                cloud: [ 'AWS', 'AZURE', 'GCP' ]
```

Comment:
the test matrix becomes gigantic (231 checks for this branch) - let's discuss what versions/operating systems/cloud providers/thin-fat combination we want to finally have

I think that for linux we can run full matrix (resulting in 3 * 3 * 2 builds), but for other OS we can run one cloud provider build on one version e.g. Windows AWS with java 8, Windows GCP with java 11 and Windows Azure with java 17 - the same with MacOS
thin-jar builds we can skip for Windows and MacOS

Reply:
Updated test matrix with proposal, this reduce the number of checks to 96( 72 on Linux and 12 for Mac and Windows each)

Reply:
Do we want to run for the old-driver all configurations or limited to AWS?

Reply:
Let's not run old driver on other clouds